### ISSUE TITLE
Microsoft.Windows.SDK.NET.Ref error coming  when debugging WinApp SDK

### ISSUE THREAD

**Issue by Darsh0307** [unoplatform/uno.templates#951]
### Current behavior

When creating an application with the 5.5 dev and running with WinApp SDK as the debugging target then in those cases there is a wired error coming asking us to change the Windows.SDK.NET , telling us to add a new PropertyGroup in the .csproj file. The error is coming with  5.5 dev version only.

Here is the message in error list. 

![Screenshot 2024-09-11 at 3 27 36 PM](https://github.com/user-attachments/assets/11cf7751-c9e5-4e02-a538-3a4c72b635ba)

Here is the result of `Build` in output tab.  

```
rogram Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2412,5): warning MSB3277:           Project file item includes which caused reference "C:\Users\DarshPatel\.nuget\packages\uno.winui\5.5.0-dev.227\lib\net8.0-windows10.0.19041.0\Uno.UI.Toolkit.dll".
1>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2412,5): warning MSB3277:             C:\Users\DarshPatel\.nuget\packages\uno.winui\5.5.0-dev.227\lib\net8.0-windows10.0.19041.0\Uno.UI.Toolkit.dll
1>C:\Users\DarshPatel\.nuget\packages\microsoft.windowsappsdk\1.6.240829007\buildTransitive\Microsoft.WindowsAppSDK.targets(76,9): error : This version of the Windows App SDK requires Microsoft.Windows.SDK.NET.Ref 10.0.19041.38 or later.
1>C:\Users\DarshPatel\.nuget\packages\microsoft.windowsappsdk\1.6.240829007\buildTransitive\Microsoft.WindowsAppSDK.targets(76,9): error :     Please update to .NET SDK 6.0.134, 6.0.426, 8.0.109, 8.0.305 or 8.0.402 (or later).
1>C:\Users\DarshPatel\.nuget\packages\microsoft.windowsappsdk\1.6.240829007\buildTransitive\Microsoft.WindowsAppSDK.targets(76,9): error :     Or add a temporary Microsoft.Windows.SDK.NET.Ref reference which can be added with:
1>C:\Users\DarshPatel\.nuget\packages\microsoft.windowsappsdk\1.6.240829007\buildTransitive\Microsoft.WindowsAppSDK.targets(76,9): error :         <PropertyGroup>
1>C:\Users\DarshPatel\.nuget\packages\microsoft.windowsappsdk\1.6.240829007\buildTransitive\Microsoft.WindowsAppSDK.targets(76,9): error :             <WindowsSdkPackageVersion>10.0.19041.38</WindowsSdkPackageVersion>
1>C:\Users\DarshPatel\.nuget\packages\microsoft.windowsappsdk\1.6.240829007\buildTransitive\Microsoft.WindowsAppSDK.targets(76,9): error :         </PropertyGroup>
1>C:\Users\DarshPatel\.nuget\packages\microsoft.windowsappsdk\1.6.240829007\buildTransitive\Microsoft.WindowsAppSDK.targets(76,9): error :
1>Done building project "blankCLI.csproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
========== Build completed at 3:21 PM and took 00.925 seconds ==========

```

### Expected behavior

It should build the app without having any error when debugging with WinApp SDK as debugging target.

### Affected platforms

Windows (WinAppSdk)

### Template Host

dotnet new

### Host OS

Windows

### Template Options

1. The App was a blank project created using dotnet new templates. 
 Here is the command used to create using blank project. `dotnet new unoapp -preset=blank -o test`
2. Select `WinApp SDK` as the debugging target. 
3. Try debugging it 
4. Open the builds in output tab. 

### Template Version

```shell
PS C:\Users\DarshPatel> dotnet new details Uno.templates
Uno.Templates
   Package version: 5.5.0-dev.7
   Reserved: True
   Details: Templates for creating Uno Platform apps and libraries
   Source Feed: nuget.org [https://api.nuget.org/v3/index.json]
   Authors:
      Uno Platform
   Owners:
      https://nuget.org/profiles/unoplatform
   License Metadata:
      License: Apache-2.0
   License Expression: https://licenses.nuget.org/Apache-2.0
      License Url: https://www.nuget.org/packages/Uno.Templates/5.5.0-dev.7/license
      Repository Url: https://github.com/unoplatform/uno.templates
PS C:\Users\DarshPatel>
```

### Anything else we need to know?

Visual Studio Version:- 17.11.3

**Comment by Darsh0307** [comment]
[blankCLI.zip](https://github.com/user-attachments/files/16969744/blankCLI.zip)

**Comment by jeromelaban** [comment]
There is no fix available for this. If you do not see any related change, the issue is likely still present.

**Comment by Darsh0307** [comment]
No worries , Sounds good @jeromelaban

**Comment by Darsh0307** [comment]
@jeromelaban  

This issue is also coming in 5.4.2 also when creating the project with CLI. 

Uno.sdk / Uno.templaets :- 5.4.2

cc @jhanvi03  @agneszitte

**Comment by jeromelaban** [comment]
We're not able to fix issue at this time, as it breaks for multiple reasons (https://github.com/mono/SkiaSharp/issues/2999). Make the modification manually to get around it by adding `<WindowsSdkPackageVersion>10.0.19041.38</WindowsSdkPackageVersion>`.

**Comment by jeromelaban** [comment]
Fixed in 5.4.5 and later (https://github.com/unoplatform/uno.templates/commit/74a36bf49f0903ec66031d7f60bdcf7138268164)
